### PR TITLE
onxx model load and some updates

### PR DESCRIPTION
**Pull request by MandeepSinghthakur**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -47,3 +48,37 @@ function processNode(node) {
...
+  // add inputs here 
+  var inputs =  new Tensor(new Float32Array([1.0, 2.0, 3.0, 4.0]), "float32", [2, 2])
+  return inputs
+}
+
+function run_model() {
+  console.log('running_')
+   const myOnnxSession = new onnx.InferenceSession();
+      // load the ONNX model file
+      myOnnxSession.loadModel("https://drive.google.com/file/d/1sPQFC6arqwT_IwSNW-q4J9bvhCrECwvK/view?usp=sharing").then(() => {
```
Comment (trtm) [inline main.js]: Maybe we should commit a basic model to the repo?

Comment (MandeepSinghthakur) [reply main.js]: @trtm Thats what should be ideal place but i wasn't able to load model from directory within so used drive link. gave 404 to me.

Comment (trtm) [reply main.js]: We will need to find a way to load it from within then.

Comment (MandeepSinghthakur) [reply main.js]: found a way to add  resource to manifest and load the url: 
"web_accessible_resources": [
    "ml/*.onxx"
  ]
going to open a PR shortly.

**Pull request merged**